cirq-google/cirq_google/json_resolver_cache.py

```diff
@@ -24,7 +24,6 @@ def _class_resolver_dictionary() -> Dict[str, ObjectFactory]:
     from cirq_google.devices.known_devices import _NamedConstantXmonDevice
 
     return {
-        '_NamedConstantXmonDevice': _NamedConstantXmonDevice,
```

Comment:
The `.json` and `.repr` files should be deleted once the objects are actually removed from the public interface. On deprecation, we should just add an entry to `deprecated = {}` dictionary. 

See https://quantumai.google/cirq/dev/serialization#deprecating_a_serializable_value

Reply:
Fair enough. I guess you weren't a fan of the "They shouldn't have been there in the first place" argument.